tests/e2e/actions.go

```diff
@@ -250,6 +258,125 @@ func (tr Chain) submitTextProposal(
 	tr.waitBlocks(action.Chain, 1, 10*time.Second)
 }
 
+type UpdateConsumerChainAction struct {
+	Chain               ChainID
+	From                ValidatorID
+	ConsumerChain       ChainID
+	SpawnTime           uint
+	InitialHeight       clienttypes.Height
+	DistributionChannel string
+	TopN                uint32
+	ValidatorsPowerCap  uint32
+	ValidatorSetCap     uint32
+	Allowlist           []string
+	Denylist            []string
+	MinStake            uint64
+	AllowInactiveVals   bool
+	NewOwner            string
```

Comment:
Could we move `NewOwner` directly after `ConsumerChain` to respect the order as seen in `MsgUpdateConsumer`. Similarly for `InitialHeight` and `SpawnTime` (order of `ConsumerInitializationParameters`).